### DISCUSSION TITLE
Invite only SignUp [Apr 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5525] [Discord Threads]
> 🙋‍♂️ **Rahim Dobani**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### You can disable from account settings (admin only option)  
![image0](https://cdn.discordapp.com/attachments/1086223810119868477/1086249237634895882/Screenshot_2023-03-17_at_4.55.47_PM.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### You will still see a signUp button, but it wouldn't allow you to signUp though  
![image0](https://cdn.discordapp.com/attachments/1086223810119868477/1086249404069052496/Screenshot_2023-03-17_at_4.55.17_PM.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1086223810119868477)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
